hello-data-deployment/docker-compose/base/jupyterhub/hello_data_scripts.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+import psycopg2
+from requests.auth import HTTPBasicAuth
```

Comment:
I think this script for connecting the database and other potential libraries `hello_data_scripts.py` would need to go into a PyPi Package. For a short term, should we add it to the side car as well, as we need to map external things into the jupyterhub already (postgres user, tornado/keycloak settings, etc)?